src/Service.Host/client/src/components/Workstations.js

```diff
@@ -22,18 +28,22 @@ function Workstations() {
             setCitCodeSearchTerm(newValue);
         }
     };
+    const auth = useAuth();
+    const token = auth.user?.access_token;
 
     const {
         send: getWorkStations,
         workStationsLoading,
         result: workStationsResult
-    } = useGet(itemTypes.workStations.url);
+    } = useGet(itemTypes.workStations.url, true);
 
     const [hasFetched, setHasFetched] = useState(false);
+    const [hasPermission, setHasPermission] = useState(false);
```

Comment:
i dont think we need an extra piece of react state for this. It'll just be a constant depending on whether the application state response contains the create link